util/test/veOcean/test_rewards.py

```diff
@@ -60,41 +61,51 @@ def test_rewards():
     veOCEAN.create_lock(toBase18(100.0), david_lock_time, {"from": david})
     
     #wait 2 days and OPF adds rewards
-    chain.sleep(DAY)
-    chain.mine()
-    chain.sleep(DAY)
-    chain.mine()
-    OCEAN.transfer(fee_distributor.address, toBase18(opffees), {"from": accounts[0]})
-    fee_distributor.checkpoint_token()
-    fee_distributor.checkpoint_total_supply()
+    #chain.sleep(DAY)
+    #chain.mine()
+    #chain.sleep(DAY)
+    #chain.mine()
+    #OCEAN.transfer(fee_distributor.address, toBase18(opffees), {"from": accounts[0]})
+    #fee_distributor.checkpoint_token()
+    #fee_distributor.checkpoint_total_supply()
```

Comment:
If all this code is commented out, why have it at all?

Reply:
fixed